typescript/packages/lattice-client-core/src/controllers/providers.ts

```diff
@@ -54,6 +54,21 @@ class ProvidersController extends BaseController {
       JSON.stringify(body),
     );
   }
+
+  /**
+   * Retrieves a list of all providers currently running in the lattice
+   * @returns a list of providers
+   */
+  async list(): Promise<ControlResponse<Record<string, WasmCloudProvider>>> {
```

Comment:
I know you had a comment about this coming from the bucket. It's definitely not ideal since the bucket data is not a public API, but it's fine until we get some new topics that give us the data we need.

Reply:
Yeah, you are totally correct, it does not belong here and I we are changing the public API's signature. I've seen that the package is pre 1.x.x so we might get away even removing this. It might worth to review an alternative approach before 1.0.0. But thanks for the heads up, really good to see that you are embracing the SemVer contracting.